packages/xo-web/src/common/intl/messages.js

```diff
@@ -452,7 +452,8 @@ const messages = {
   migrateBackupSchedule: 'Migrate to the new backup system',
   migrateBackupScheduleMessage:
     'This will convert the legacy backup job to the new backup system. This operation is not reversible. Do you want to continue?',
-  runBackupNgJobConfirm: 'Are you sure you want to run {name} ({id})?',
+  runBackupJobConfirm:
+    'The {name} ({id}) job will backup {nVms, number} VM{nVms, plural, one {} other {s}}. Are you sure you want to run it?',
```

Comment:
Ideally, we shouldn't change the meaning of a message without also changing its ID.

Reply:
Ok, I will keep the ID and I don't change the meaning of the message.